src/styles/variables.less

```diff
@@ -13,10 +13,11 @@
 @locked: #000000;
 @incomplete: #ff9100;
 @questAtLocation: #55ff00;
-@uncaughtPokemon: #3498db;
+@uncaughtPokemon: #d8b0db;
```

Comment:
If we change this, we want an update function for all the players who are still using default colors. Just something like `if (settings.NameOfUncauftPokemonMapColorSettings == '#3498db') settings.NameOfUncauftPokemonMapColorSettings = '#d8b0db'`

Reply:
Removed boolean setting, set new setting as white. Should not need update function.